### PR TITLE
Adding config file as a parameter to entrypoint.

### PR DESCRIPTION
**Pull request by jubeira**
See #10; needs https://github.com/osrf/polymer-ros-rviz/pull/15.

To specify a configuration, use `config_file` parameter; e.g:
```
roslaunch rvizweb rvizweb.launch config_file:=/path/to/config/file
```

The configuration file is a JSON just like the one that is displayed in the popup after clicking `LOAD CONFIG` at the left panel.

### CODE REVIEW

File: config/configuration [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (chapulina) [inline config/configuration]: Should we give this file an extension? `.json`?

File: launch/rvizweb.launch [hunk 2]
```diff
@@ -32,6 +32,11 @@
   <!-- RGB image topic for depthcloud_encoder. -->
   <arg name="rgb_topic" default="/camera/rgb/image_raw"/>
 
+  <!-- Configuration. -->
+  <arg name="config_file" default="$(find rvizweb)/config/configuration"/>
+
+  <param name="/rvizweb/global_config" type="str" textfile="$(find rvizweb)/config/configuration"/>
```
Comment (chapulina) [inline launch/rvizweb.launch]:
```suggestion
  <param name="/rvizweb/global_config" type="str" textfile="$(arg config_file)"/>
```

**Review by chapulina — CHANGES_REQUESTED** [review]
Works for me with the proposed changes. Mind also adding a section to the README with an example of how to use a custom config file? Thanks!

**Comment by jubeira** [comment]
Comments addressed; thanks! Added new section to README as well.
@chapulina PTAL.

**Review by chapulina — APPROVED** [review]

**Pull request merged**